src/main.js

```diff
@@ -25,49 +25,55 @@ export default class LevelSelector extends UICorePlugin {
     }
   }
 
+  get __container() {
```

Comment:
We can just call this getter as `container`, no?

Reply:
Sure, it was prefixed by default to distinguish it as "special" getter; let's refactor them.